validate/validate.go

```diff
@@ -256,28 +256,30 @@ func (v *Validator) CheckProcess() (msgs []string) {
 		}
 	}
 
-	for _, capability := range process.Capabilities {
-		if err := CapValid(capability, v.HostSpecific); err != nil {
-			msgs = append(msgs, fmt.Sprintf("capability %q is not valid, man capabilities(7)", capability))
+	if v.spec.Platform.OS == "linux" {
+		for _, capability := range process.Capabilities {
```

Comment:
> But it is not clear the value of other systems, it can not be verified.

I agree, and think somebody with more weight than me should bring this up among the runtime-spec maintainers before they cut 1.0 (I [tried][1], but didn't have much success).

> My idea is to verify the linux system under the capabilities of the special value, not the entire capabilities field validation…

That's fine, but I don't think we should be silently passing on non-Linux OSes.  Does [moving the Linux-specific cap validation into `CheckLinuxCapabilities` and raising not-implemented-yet errors for non-Linux configs][2] sound like a bad idea to you?

[1]: https://github.com/opencontainers/runtime-spec/pull/673/files#r99353136
[2]: https://github.com/opencontainers/runtime-tools/pull/350#discussion_r107457088